### PR TITLE
Convert integer to string

### PR DESCRIPTION
**Pull request by shellhub**
#### Description of Change

Convert integer number to string 

#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#New-File-Name-guidelines)
- [x] Added tests and example, test must pass
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes: 

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/C/pull/586"></a>

### CODE REVIEW

File: conversions/int_to_string.c [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    {
+        char t = dest[i];
+        dest[i] = dest[len - 1 - i];
+        dest[len - 1 - i] = t;
+    }
+    dest[len] = '\0';
+    return dest;
+}
+
+/** Test function */
```
Comment (kvedala) [inline conversions/int_to_string.c:46]:
```suggestion
/** Test function
 * @returns `void`
 */
```

File: conversions/int_to_string.c [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+void test()
```
Comment (kvedala) [inline conversions/int_to_string.c:47]:
```suggestion
static void test()
```

**Comment by shellhub** [comment]
I have updated code as you pointed. But error still exists. @kvedala

**Comment by kvedala** [comment]
> I have updated code as you pointed. But error still exists. @kvedala

see shellhub#1

**Pull request merged**